docs/getting_started.rst

```diff
@@ -1,16 +1,119 @@
 Getting Started
 ===============
 
+Production setup
+----------------
+
+`Edx-enterprise` is developed as a pluggable application for the edx-platform and can't be used outside of edx-platform.
+
+`Edx-enterprise` is shipped with default edx-platform setup (see `edx-platform requirements file`_), so new installs
+should already have it set up and enabled. However, if you're running an older version (i.e. pre-Ficus) of edx-platform,
+manual setup must be performed. In `edxapp` virtualenv (`$current_release` is |release|)\ [#f1]_
+
+.. code-block:: bash
+
+    $ cd /edx/app/edxapp/edx-platform
+    $ pip install edx-enterprise==$current_release
+
+
+Than, add `edx-enterprise` to `INSTALLED_APPS` or `OPTIONAL_APPS` (see `lms/env/common.py`_ as an example) and
+run migrations.
+
+.. code-block:: bash
+
+    $ paver update_db
+    # Or use a more down-to-the-root command (replace aws with your version of config)
+    $ ./manage.py lms migrate --settings=aws
+
+.. _edx-platform requirements file: https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt
+.. _lms/env/common.py: https://github.com/edx/edx-platform/blob/488467883409b9d06df3e0ccfa55fe8e8a9903b3/lms/envs/common.py#L2678-L2681
+
+Local development
+-----------------
+
+Despite `edx-enterprise` can't be used outside edx-platfrom, some development process steps are intended to be
+performed in a standalone virtual environment.
+
 If you have not already done so, create/activate a `virtualenv`_. Unless otherwise stated, assume all terminal code
 below is executed within the virtualenv.
 
 .. _virtualenv: https://virtualenvwrapper.readthedocs.org/en/latest/
 
 
 Install dependencies
---------------------
+^^^^^^^^^^^^^^^^^^^^
+
 Dependencies can be installed via the command below.
 
 .. code-block:: bash
 
     $ make requirements
+
+
+Run tests
+^^^^^^^^^
+
+This application uses `tox`_ to run tests under multiple python and Django version. In addition, the following test
+utilities are used:
```

Comment:
@bradenmacdonald I've thought about it as well and decided to land this PR first, so that I don't have to make same changes to two PRs.